src/main.cpp

```diff
@@ -942,7 +1020,7 @@ int main(int argc, char *argv[]) {
 
   if (intel_log_power)
   {
-    h5_write_single<float>(out_name, "/housekeeping/intel/TDP", rapl->get_TDP(),
+    h5_write_single<uint32_t>(out_name, "/housekeeping/intel/TDP", rapl->get_TDP(),
```

Comment:
Other power data are stored as `float`. Should we really use two different data types here?

Reply:
No, float is ok, the drivers just return ints, so it was just the intuitive habit to change everything to the same datatype without thinking...

Reply:
Okay. Will you change it back to `float` in that case?